app.py

```diff
@@ -325,10 +328,21 @@ def open_curr_quest(quest_id):
     return render_template('curr_task_template.html', quest=quest)
 
 
+# Route to handle `Report Quest` Button
+@login_required
+@app.route('/report_quest/<curr_quest_id>')
+def report_quest(curr_quest_id):
+    quest = Quest.query.get(curr_quest_id)
+    reported_quest = ReportedQuest(quest_id=quest.quest_id, report_status='Not Resolved')
+    db.session.add(reported_quest)
+    db.session.commit()
+    return 'reported?'
+
 # Route to handle solution submission
 @login_required
 @app.route('/submit-solution', methods=['POST'])
 def submit_solution():
+    print(ReportedQuest.query.all())
```

Comment:
that print is not needed in the `submit-solution` route

Reply:
Last commit: 330d0f1
- The print was just for testing the functionality (I forgot to remove it the other day)
- `print(ReportedQuest.query.all())` is removed